filelink.go

```diff
@@ -53,7 +53,7 @@ func (c *FileLink) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
-// FileLinkList is a list of file links as retrieved from a list endpoint.
+// FileLinkList is a list object for file links.
```

Comment:
I think we should keep that comment since we tried to make it consistent across packages

Reply:
Makes sense. Reverted to the original comment.